pkgs/applications/audio/cheesecutter/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, lib, fetchFromGitHub, fetchpatch
+, acme, ldc, makeWrapper
+, SDL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cheesecutter";
+  version = "unstable-2019-12-06";
+
+  src = fetchFromGitHub {
+    owner = "theyamo";
+    repo = "CheeseCutter";
+    rev = "6b433c5512d693262742a93c8bfdfb353d4be853";
+    sha256 = "1szlcg456b208w1237581sg21x69mqlh8cr6v8yvbhxdz9swxnwy";
+  };
+
+  buildInputs = [ SDL ];
+
+  nativeBuildInputs = [ ldc acme makeWrapper ];
+
+  patches = [
+    ./0001-fix-impure-build-date-display.patch
+  ];
+
+  makefile = "Makefile.ldc";
+
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  postInstall = ''
+    cd $out
+    mkdir -p {bin,share/cheesecutter}
+    mv -t bin ccutter ct2util
+    mv -t share/cheesecutter example_tunes
+
+    for exe in $out/bin/{ccutter,ct2util}; do
+      wrapProgram $exe \
+        --set LD_LIBRARY_PATH "${lib.makeLibraryPath buildInputs}"
+    done
+  '';
+
+  meta = with lib; {
+    description = "A tracker program for composing music for the SID chip.";
+    homepage = "https://github.com/theyamo/CheeseCutter/";
+    license = licenses.gpl2;
+    platforms = platforms.all;
```

Comment:
Always unsure what to pick here, since it doesn't evaluate on AArch64 because of LDC. I'll adjust it to LDC's platforms later.